lxd/db.go

```diff
@@ -421,7 +424,13 @@ func dbImageGet(d *Daemon, name string, public bool) (*shared.ImageBaseInfo, err
 	if public {
 		q = q + " AND public=1"
 	}
-	err = d.db.QueryRow(q, (name+"%")).Scan(&image.Id, &image.Fingerprint, &image.Filename, &image.Size, &image.Public)
+
+	arg2 = []interface{}{&image.Id, &image.Fingerprint, &image.Filename, &image.Size, &image.Public}
+	err = shared.DbQueryRowScan(d.db, q, arg1, arg2)
+	err = shared.DbQueryRowScan(d.db, q, arg1, arg2)
```

Comment:
On Mon, Mar 16, 2015 at 08:51:01PM -0700, St�phane Graber wrote:
> > @@ -421,7 +424,13 @@ func dbImageGet(d *Daemon, name string, public bool) (*shared.ImageBaseInfo, err
> >  	if public {
> >  		q = q + " AND public=1"
> >  	}
> > -	err = d.db.QueryRow(q, (name+"%")).Scan(&image.Id, &image.Fingerprint, &image.Filename, &image.Size, &image.Public)
> > +
> > +	arg2 = []interface{}{&image.Id, &image.Fingerprint, &image.Filename, &image.Size, &image.Public}
> > +	err = shared.DbQueryRowScan(d.db, q, arg1, arg2)
> > +	err = shared.DbQueryRowScan(d.db, q, arg1, arg2)
> 
> Hmm, what?

Some people like to shout, I like to calmly repeat myself.